tests/implementor/test_run.py

```diff
@@ -0,0 +1,103 @@
+import uuid
+from time import sleep
+from unittest.mock import patch
+
+import pytest
+
+from src import implementor as impl
+from src.communicator.communicator import Communicator
+from src.implementor.models import Run
+
+
+class TestRunImplementor:
+    def test_get_all_simulation_ids(self, token, empty_simulation_configuration):
+        run = Run.create(simulation_configuration=empty_simulation_configuration)
+        result, status = impl.run.get_all_run_ids({}, token)
+        assert status == 200
+        assert str(run.id) in result
+
+    def test_get_all_simulation_ids_option(
+        self,
+        token,
+        empty_simulation_configuration,
+        another_empty_simulation_configuration,
+    ):
+        run = Run.create(simulation_configuration=empty_simulation_configuration)
+        another_run = Run.create(
+            simulation_configuration=another_empty_simulation_configuration
+        )
```

Comment:
Should these runs be created in a fixture?

Reply:
I'm not a big fan of fixtures using this way. I have the rule, that I use fixtures as infrastructure I can use. But I don't do assertions with them, because something can change an break in tests, you know?